personalized_nlp/models/baseline.py

```diff
@@ -4,10 +4,12 @@
 
 class Net(nn.Module):
 
-    def __init__(self, output_dim=2, text_embedding_dim=768, **kwargs):
```

Comment:
I think that `max_seq_len` parametr can be removed, it is not used anywhere in the class. In fact for baseline model which does not finetune embeddings it is not needed because these embeddings are calculated before in the datamodule.